runtime/query_test.go

```diff
@@ -60,6 +59,21 @@ func TestPopulateParameters(t *testing.T) {
 				"wrapper_bool_value":    {"true"},
 				"wrapper_string_value":  {"str"},
 				"wrapper_bytes_value":   {"Ynl0ZXM"},
+				"map_value[key]":        {"value"},
+				"map_value2[key]":       {"-2"},
+				"map_value3[-2]":        {"value"},
+				"map_value4[key]":       {"-1"},
+				"map_value5[-1]":        {"value"},
+				"map_value6[key]":       {"3"},
+				"map_value7[3]":         {"value"},
+				"map_value8[key]":       {"4"},
+				"map_value9[4]":         {"value"},
+				"map_value10[key]":      {"1.5"},
+				"map_value11[1.5]":      {"value"},
+				"map_value12[key]":      {"2.5"},
+				"map_value13[2.5]":      {"value"},
+				"map_value14[key]":      {"true"},
+				"map_value15[true]":     {"value"},
```

Comment:
Can you add some examples where you populate multiple map values at the same time?